tests/vm_test.go

```diff
@@ -967,6 +967,44 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 	})
 
+	DescribeTable("[release-blocker][test_id:299][test_id:264]should create and delete a VM using all supported API versions", decorators.Conformance, func(version string) {
+		vm := libvmi.NewVirtualMachine(libvmifact.NewGuestless(), libvmi.WithRunStrategy(v1.RunStrategyAlways))
+		vm.APIVersion = version
+
+		By("Creating VM")
+		vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(nil)).Create(context.Background(), vm, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Waiting for VMI to start")
+		Eventually(ThisVMIWith(vm.Namespace, vm.Name), 120*time.Second, 1*time.Second).Should(BeRunning())
```

Comment:
```suggestion
		Eventually(ThisVMIWith(vm.Namespace, vm.Name)).WithTimeout(120*time.Second).WithPolling(1*time.Second).Should(BeRunning())
```

Reply:
@0xFelix Hi, are you going to take this change?

Reply:
Do we have a policy on it?